### PR TITLE
Simplify webconsole hud

### PR DESCRIPTION
**Pull request by jasonLaster**
Some additional cleanups and a couple of fixes

### CODE REVIEW

File: src/devtools/client/webconsole/actions/toolbox.js [hunk 1]
```diff
@@ -26,38 +27,47 @@ function unHighlightDomElement(grip) {
...
+    }
   };
 }
 
-export function openNodeInInspectorgrip() {
-  return ({ hud }) => {
-    hud.openNodeInInspector(grip);
-  };
-}
+export function openNodeInInspector(valueFront) {
```
[the remark this answers is not in the document]
Comment (jaril) [reply src/devtools/client/webconsole/actions/toolbox.js:41]: Do we have to change https://github.com/RecordReplay/devtools/blob/cebb7ffa910351227c153f330b84b2fbf750c0f4/src/devtools/client/webconsole/utils/object-inspector.js#L35 ?

File: src/devtools/client/webconsole/components/FilterBar/ConsoleSettings.js [hunk 2]
```diff
@@ -23,12 +23,10 @@ const { FILTERS } = require("devtools/client/webconsole/constants");
 class ConsoleSettings extends Component {
   static get propTypes() {
     return {
-      dispatch: PropTypes.func.isRequired,
```
Comment (jaril) [inline src/devtools/client/webconsole/components/FilterBar/ConsoleSettings.js:26]: Are we not passing in dispatch anymore? It looks like it's still being used in the rest of the file and is still being passed from the parent (<FilterBar />)

File: src/devtools/client/webconsole/components/FilterBar/FilterBar.js [hunk 3]
```diff
@@ -37,16 +37,13 @@ class FilterBar extends Component {
   static get propTypes() {
     return {
       closeButtonVisible: PropTypes.bool,
-      closeSplitConsole: PropTypes.func,
-      dispatch: PropTypes.func.isRequired,
```
Comment (jaril) [inline src/devtools/client/webconsole/components/FilterBar/FilterBar.js:41]: Looks like we still need this for the child (<ConsoleSettings />).

Comment (jaril) [reply src/devtools/client/webconsole/components/FilterBar/FilterBar.js:41]: Now that I looked into it, the <FilterBar />'s parent doesn't appear to be passing in a dispatch prop. Am I missing something here?

https://github.com/RecordReplay/devtools/blob/cebb7ffa910351227c153f330b84b2fbf750c0f4/src/devtools/client/webconsole/components/App.js#L154-L158

File: src/devtools/client/webconsole/components/Input/JSTerm.js [hunk 4]
```diff
@@ -117,7 +111,7 @@ class JSTerm extends Component {
     this.autocompletePopup = null;
 
     EventEmitter.decorate(this);
-    webConsoleUI.jsterm = this;
+    window.jsterm = this;
```
Comment (jaril) [inline src/devtools/client/webconsole/components/Input/JSTerm.js:114]: So weird seeing this but I get it 😬

File: src/devtools/client/webconsole/actions/toolbox.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/devtools/client/webconsole/actions/toolbox.js:41]: I think it should work, but we should test it :)

File: src/devtools/client/webconsole/components/FilterBar/FilterBar.js (hunk 3 above)
Comment (jasonLaster) [reply src/devtools/client/webconsole/components/FilterBar/FilterBar.js:41]: Good catch. I added dispatch

File: src/devtools/client/webconsole/components/Input/JSTerm.js (hunk 4 above)
Comment (jasonLaster) [reply src/devtools/client/webconsole/components/Input/JSTerm.js:114]: yeah...

**Pull request merged**